graph/src/data/subgraph/schema.rs

```diff
@@ -164,6 +203,35 @@ impl SubgraphVersionEntity {
         entity.set("createdAt", self.created_at);
         vec![set_entity_operation(Self::TYPENAME, id, entity)]
     }
+
+    pub fn query_from_deployment(id: &SubgraphDeploymentId) -> EntityQuery {
+        EntityQuery::new(SUBGRAPHS_ID.to_owned(), Self::TYPENAME)
+            .filter(EntityFilter::new_equal("deployment", id.to_string()))
+    }
```

Comment:
`EntityQuery::new(SUBGRAPHS_ID.to_owned(), Self::TYPENAME)` can be simplified to `Self::query()`